source/jormungandr/tests/departure_board_tests.py

```diff
@@ -397,6 +397,21 @@ def test_vj_comment(self):
                 # assert that if it's the 'all' vj there is no note
                 assert(len(vj_link) == 0 or len(notes_link) == 0)
 
+    def test_line_closed(self):
+        """
+        Check the ResponseStatus when the line is closed
+        """
+        response_1 = self.query_region("lines/C/stop_schedules?from_datetime=20120614T0830&duration=600")
+        response_2 = self.query_region("lines/D/stop_schedules?from_datetime=20120614T0700&duration=600")
+        import json
+        print(json.dumps(response_2))
```

Comment:
you should remove that, and you can have a trace using self.query_region("lines/D/stop_schedules?from_datetime=20120614T0700&duration=600", display=True)``